src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,17 @@
 #include "BatteryStateOfChargeService.h"
+#include <stdio.h>
 
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    double ampHourTotal;
+    double ampHourUsed;
```

Comment:
Oh i see, so namespace is a static field then. I should've done more research on what a namespace is, had put them there "because it worked" and because I had trouble declaring it as private. I'll keep that in mind thanks! :)